pages/portfolio.html

```diff
@@ -1,12 +1,97 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <link href="styles/portfolio.css" rel="stylesheet">
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>&hearts;portfolio &hearts;</title>
 </head>
+
 <body>
+  <header>
+    <h1>&hearts;portfolio&hearts;</h1>
+    <nav>
+      <div class="dropdown">
+          <span>navigation:</span>
+          <div class="dropdown-content">
+              <a href="index.html" >home</a>
+              <br><a href="about.html">about</a>
+              <br><a href="portfolio.html">portfolio</a>
+            </div>
+        </div>
+      </nav>
+      <br>
+  </header>
   
+  <main>
+    <section class="portfolio-a">
+      <img src="/Users/shelby_faulconer/ada/projects/personal-portfolio-site/assets/tasklist2.png" alt="screen shot of Python project, Task List API">
+      <h3>Task List API</h3>
+      <p class="project-description">
+      A task list API build using Python SQL and Flask, integrated with Slack with a bot to announce when tasks are marked complete,
+      and deployed through Heroku.<br>
+      </p>
+      <span>
+      <br>
+      <a href="https://github.com/faulconershelby/task-list-api" class="project-link">Github</a>
+      &hearts;
+      <a href="https://bujo-goal-bot.herokuapp.com/tasks" class="project-link">Heroku</a>
+      <br>
```

Comment:
We should indent the content of `span` to help show the parent/child relationship.